src/content/docs/devtools/Docker/docker-case-study.md

```diff
@@ -0,0 +1,153 @@
+| Attribute  | Details                                                      |
+|------------|--------------------------------------------------------------|
+| Title      | Docker case study                                            |
+| Author     | Eirik Michielsen <EM-90>                                     |
+| Tags       | docker, docker image, containerize, container, deployment, development, environment |
+
+
+### Introduction
+
+In this case study I will be writing about the benefit of using Docker by focusing on Docker image and containerizing applications.
```

Comment:
Using the word I in this kind of write makes it look subjective. Whereas a software-related article should be objective. Please, fix the tone of the write to sound third-person. @EM-90

Reply:
Yes, I will make the change